src/on-behalf-of/queries/use-get-phone-by-account-id-query.tsx

```diff
@@ -0,0 +1,13 @@
+import {getPhoneNumberFromId} from '@atb/api/profile';
+import {useQuery} from '@tanstack/react-query';
+
+const ONE_MINUTE = 1000 * 60;
+
+export const useGetPhoneByAccountIdQuery = (accountId: string) =>
+  useQuery({
+    queryKey: ['getPhoneByAccountId', accountId],
+    queryFn: () => getPhoneNumberFromId(accountId),
+    enabled: accountId !== undefined && accountId !== '',
```

Comment:
Good that the request is skipped if empty or nil values comes to accountId

Reply:
updated